templates/header-right.mustache

```diff
@@ -5,7 +5,8 @@
 	    	<ul class="dropdown-menu">
 	      		<li><a href="http://bitovi.com">Bitovi.com</a></li>
 	      		<li><a href="http://bitovi.com/blog/">Blog</a></li>
-	      		<li><a href="http://bitovi.com/consulting/">Consulting</a></li>
+	      		<li><a href="http://bitovi.com/design/">Design</a></li>
+	      		<li><a href="http://bitovi.com/development/">Development</a></li>
```

Comment:
These should be updated to match the destination after redirecting, e.g. https://www.bitovi.com/development